ui/src/reusable_ui/constants/index.ts

```diff
@@ -0,0 +1,21 @@
+export const COMPONENT_COLORS = {
```

Comment:
what do you think about using enums for these?

Reply:
Done!